packages/wrapped-keys/src/lib/constants.ts

```diff
@@ -0,0 +1,31 @@
+const CHAIN_ETHEREUM = 'ethereum';
+const LIT_PREFIX = 'lit_';
+export const NETWORK_EVM = 'evm';
+export const NETWORK_SOLANA = 'solana';
+export type Network = typeof NETWORK_EVM | typeof NETWORK_SOLANA;
+
+// Update the endpoint to Wrapped Key project endpoint
+const ENCRYPTED_PRIVATE_KEY_ENDPOINT =
+  'https://8wugrwstu1.execute-api.us-east-2.amazonaws.com/encrypted';
+
+const LIT_ACTION_CID_REPOSITORY = Object.freeze({
```

Comment:
can we have some comments briefly what they do and what are the expected inputs (think a function params) and output (think return type)?

Reply:
Added JsDocs to the Lit Action files